requirements/prod-requirements.txt

```diff
@@ -405,7 +405,7 @@ requests==2.25.1
     #   twilio
 requests-oauthlib==1.3.0
     # via -r base-requirements.in
-requests_toolbelt==0.9.1
+requests-toolbelt==0.9.1
```

Comment:
I assume these changes to the txt files are just aliases for each other so it's not a real change. Curious if you have more you can share about how that change happened, i.e. whether it was due to a change on our side / within this PR, or a change to the metadata on pypi for these packages or something.

Reply:
I actually am not sure how this change happened. Even reverting it manually and re-running on `python 3.6.8` produced the same diff, so I just went with it